src/compressed_tensors/quantization/lifecycle/apply.py

```diff
@@ -103,3 +113,11 @@ def _find_first_match(value: str, targets: Iterable[str]) -> Optional[str]:
         elif target == value:
             return target
     return None
+
+
+def _infer_status(model: Module) -> Optional[QuantizationStatus]:
```

Comment:
Do we ever want to support having different status' in different parts of the model? I know its out of scope for now, but I could see eventually wanting to freeze some layers while calibrating others and this fn would break that

Reply:
was thinking through the same thing as a limitation - also since layers that are activation only can't be compressed (see todo)